### PR TITLE
Migrate package push API to GraphQL

### PR DESCRIPTION
**Pull request by nl0**
# Description

# TODO

- [x] resolve merge conflicts
- [x] [Changelog](CHANGELOG.md) entry

### CODE REVIEW

File: catalog/app/containers/Bucket/PackageDialog/PackageDialog.tsx [hunk 1]
```diff
@@ -380,35 +380,29 @@ export type SchemaFetcherRenderProps =
...
   const s3 = AWS.S3.use()
   const sentry = Sentry.use()
 
-  const slug = manifest?.workflow?.id
-
   const initialWorkflow = React.useMemo(() => {
     // reuse workflow from previous revision if it's still present in the config
-    if (slug) {
-      const w = workflowsConfig.workflows.find(R.propEq('slug', slug))
+    if (initialWorkflowId) {
```
Comment (fiskus) [inline catalog/app/containers/Bucket/PackageDialog/PackageDialog.tsx:400]: :+1:

File: catalog/app/containers/Bucket/PackageDirectoryDialog.tsx [hunk 2]
```diff
@@ -461,19 +481,16 @@ export default function PackageDirectoryDialog({
...
 
-    onExited({
-      pushed: success,
-    })
+    onExited({ pushed: success })
     if (onClose) onClose()
     setSuccess(null)
   }, [submitting, success, setSuccess, onClose, onExited])
 
+  // XXX: something's wrong here with these identical handlers
```
Comment (fiskus) [inline catalog/app/containers/Bucket/PackageDirectoryDialog.tsx:489]: I'll check this

Comment (fiskus) [reply catalog/app/containers/Bucket/PackageDirectoryDialog.tsx:489]: Filed task

**Review by fiskus — COMMENTED** [review]
Just read the code
Now I'm going to test manually

**Review by fiskus — APPROVED** [review]

**Comment by nl0** [comment]
@fiskus rebased

**Pull request merged**